site-src/concepts/security-model.md

```diff
@@ -100,12 +100,3 @@ the ReferenceGrant.
 For more information on ReferenceGrant, refer to our [detailed documentation
 for this resource](../api-types/referencegrant.md).
 
-## Advanced Concept: Limiting Namespaces Where a GatewayClass Can Be Used
```

Comment:
removed this because it is not quite true anymore with VAP, and also this is added on security-considerations now